lib/prompt.mli

```diff
@@ -1,17 +1,19 @@
-val confirm : question:('a, unit) Logs.msgf -> yes:bool -> bool
-(** Promtps the user for confirmation. [confirm ~question ~yes] uses the message
-    formatting function [question] to format and log a message with the app
-    level and wait for a yes or no answer from the user. Returns [true] for yes.
-    Defaults to yes if the user just press enter. If [yes] then just skip the
-    prompt and returns [true]. E.g.
+type default = Yes | No
```

Comment:
Not that it matters much but how about naming this type `answer` instead? I think it makes more sense.